examples/okta_inline_hook/basic_updated.tf

```diff
@@ -0,0 +1,20 @@
+resource okta_inline_hook test {
+  name    = "testAcc_replace_with_uuid"
+  status  = "INACTIVE"
+  type    = "com.okta.import.transform"
+  version = "1.0.2"
+
+  channel = {
+    type    = "HTTP"
+    version = "1.0.0"
+    uri     = "https://id-mapper.articulate.com/v1/users"
```

Comment:
And this please

Reply:
9996f7f7d908f119efd9dc949730acd335c64162

Reply:
Thank you

Reply:
For sure, good catch.